packages/mui-material/src/Button/Button.js

```diff
@@ -5,6 +5,8 @@ import clsx from 'clsx';
 import resolveProps from '@mui/utils/resolveProps';
 import composeClasses from '@mui/utils/composeClasses';
 import { alpha } from '@mui/system/colorManipulator';
+import { unstable_useId as useId } from '@mui/material/utils';
+import CircularProgress from '@mui/material/CircularProgress';
```

Comment:
This can't work

Reply:
Tackled in https://github.com/mui/material-ui/pull/44473

Reply:
MUI X had this problem multiple times in the past. As far as I remember, it effectively breaks the release, we had to do a hot fix new release. I like that they enforce this now with eslint: https://github.com/mui/mui-x/blob/340acbde76bbad3d7925f3c374cb9f23a8d7f924/.eslintrc.js#L70.